.github/workflows/smoke-test.yml

```
@@ -54,12 +54,27 @@ jobs:
           sudo rm -rf /var/lib/docker /etc/docker /run/docker* /var/lib/containerd /run/containerd*
           sudo ip link delete docker0 || :
 
-          sudo apt update
-          sudo apt install -y snapd
-          sudo snap refresh
-          sudo snap refresh --channel beta snapd
+      - name: refresh snapd
+        run: |
+          sudo snap install snapd
+          sudo snap refresh snapd
           sudo snap list
```

Refresh is unnecessary since `snap install` has just installed the latest/stable.
```suggestion
      # The image comes with a slightly outdated snapd Debian package
      - name: Install latest snapd as a snap
        run: |
          sudo snap install snapd
          sudo snap list
```